.github/workflows/cli-release.yaml

```diff
@@ -64,5 +68,6 @@ jobs:
           repo_token: "${{ secrets.GITHUB_TOKEN }}"
           automatic_release_tag: "${{ github.event.inputs.release_tag }}"
           prerelease: false
+          title: "${{ github.event.inputs.release_title }}"
```

Comment:
what happens if the user leaves `title` empty? Will this create a release with an empty title or just proceed as if no title had been passed to the action?

Reply:
I tested that, it falls back to the default, not an empty string.